shanoir-ng-datasets/src/main/java/org/shanoir/ng/download/WADODownloaderService.java

```diff
@@ -159,7 +156,7 @@ public List<String> downloadDicomFilesForURLsAsZip(final List<URL> urls, final Z
 				String name = buildFileName(subjectName, dataset, datasetFilePath, sopInstanceUID);
 				// Download and zip
 				try {
-					String zipedFile = downloadAndWriteFileInZip(url, zipOutputStream, name);
+					String zipedFile = downloadAndWriteFileInZip(url, zipOutputStream, name, url.contains(WADO_REQUEST_TYPE_WADO_RS));
```

Comment:
I think test should be directly made in downloadAndWriteFileInZip, as the tested url is in the parameters

```
// remove isMultipart parameter from method signature
private String downloadAndWriteFileInZip(String url, ZipOutputStream zipOutputStream, String name) throws ZipPacsFileException {
   ....
   this.extractDICOMZipFromMHTMLFile(responseBody,  name, zipOutputStream, url.contains(WADO_REQUEST_TYPE_WADO_RS));
   ...
}
```